Gruntfile.js

```diff
@@ -398,57 +416,48 @@ module.exports = function (grunt) {
     'bump-commit'
   ]);
 
-  grunt.registerTask('minify', [
-    'concat',
-    'uglify'
-  ]);
-
   grunt.registerTask('server', [
-    'less:development',
-    'ngconstant',
-    'sails-linker',
+    'precompile',
     'connect:server',
-    'watch:css'
+    'watch'
   ]);
 
-  grunt.registerTask('serverjs', [
-    'less:development',
-    'ngconstant',
-    'sails-linker',
-    'connect:server',
-    'watch:javascript'
+  grunt.registerTask('serverall', [
+    'server'
```

Comment:
I originally meant for `grunt server` as a way of refreshing the server and not running test incase it took ages to refresh due to hundreds of test but due to the way i've broken up the watch task it's not really possible anymore without running certain tasks unnecessarily. I'll remove it.